tests/parser/ParseContextTests.cpp

```diff
@@ -748,3 +748,25 @@ BOOST_AUTO_TEST_CASE(Test_EXIT1) {
   the extra process - that kind of feelt a bit over the top.
 */
 }
+
+
+/*
+  If there are errors the ErrorGuard destructor will print error messages and
+  call std::terminate() in the test we therefor create the instance with errors
+  using new - and let it leak.
+*/
+
+
+BOOST_AUTO_TEST_CASE(Test_ERRORGUARD) {
+    ErrorGuard el1;
+
```

Comment:
You should also check that the dump() function works, so perhaps it should be public?

Reply:
I don't know; I am a test bigot as good as any - but I have come to draw the line at making functionality public *just to be able to test it*.